plugins/osdn/factory/factory.go

```diff
@@ -3,31 +3,31 @@ package factory
 import (
 	"strings"
 
-	"github.com/openshift/openshift-sdn/plugins/osdn"
-	"github.com/openshift/openshift-sdn/plugins/osdn/api"
-
 	osclient "github.com/openshift/origin/pkg/client"
 	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/storage"
 
+	"github.com/openshift/openshift-sdn/plugins/osdn"
+	"github.com/openshift/openshift-sdn/plugins/osdn/api"
 	"github.com/openshift/openshift-sdn/plugins/osdn/ovs"
 )
 
 // Call by higher layers to create the plugin SDN master instance
-func NewMasterPlugin(pluginType string, osClient *osclient.Client, kClient *kclient.Client) (api.OsdnPlugin, error) {
-	return newPlugin(pluginType, osClient, kClient, "", "")
+func NewMasterPlugin(pluginType string, osClient *osclient.Client, kClient *kclient.Client, etcdHelper storage.Interface) (api.OsdnPlugin, error) {
```

Comment:
I also need ectd helper for vnid ranges: https://github.com/openshift/openshift-sdn/pull/303/commits/08ae21eaca31a2e10e27845f111efa0016d7f24a#diff-643475b06fe7640bd0eab5a38497f7beR34